client/src/visualization/authServerService.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+const baseUrl = 'http://localhost:3001';
```

Comment:
Question: Will this URL work also in the production environment?

Reply:
The current "production environment" only supports static hosting, so we would need a separate service for hosting the nodejs server.

In the final production environment (tkt virtual machine) we probably want an env variable for setting the API url, so that we can change the api url without changing the code.

Reply:
Hardcoded api url is good for now